core/src/components/select/select.scss

```diff
@@ -259,6 +261,19 @@ button {
   transition: opacity 150ms cubic-bezier(0.4, 0, 0.2, 1);
 }
 
+.select-wrapper-inner {
+  display: flex;
+
+  align-items: center;
+
+  overflow: hidden;
+}
+
+:host(.select-label-placement-stacked) .select-wrapper-inner,
+:host(.select-label-placement-floating) .select-wrapper-inner {
+  flex-grow: 1;
+}
```

Comment:
Added in https://github.com/ionic-team/ionic-framework/pull/28563/commits/8c0269b98832025fedb7f4f945a421bd6fca8de5. I think your comment also referenced the `.select-wrapper-inner` rule without the stacked/floating labels prefix, but I wasn't sure if that was intentional since the styles there seem self-explanatory to me. LMK if you were thinking otherwise.

Reply:
Ah yes sorry I should not have included that block. The prefixed rules was what I wanted to focus on.